cypress/support/utils/post-cards.js

```diff
@@ -0,0 +1,34 @@
+const calculateClicks = (total) => {
+  const postsPerPage = 25;
+
+  // If returning the num of clicks, subtract 1 because the first page is
+  // fully populated
+  return total <= postsPerPage ? 0 : Math.ceil(total / postsPerPage) - 1;
+};
+
+const getPostCards = () => cy.get(".post-feed").find(".post-card");
+
+const loadAndSumAllPostCards = (selector) => {
```

Comment:
```suggestion
const loadAndSumAllPostCards = (selector) => {
```
Should this be `loadAndCountAllPostCards`?

Reply:
Yes, I think `loadAndCountAllPostCards` makes more sense than the current name. I'll go in and change that now :+1: